packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -245,7 +245,8 @@ const hasMenuEveryMenuVisibilitySetToBeHidden = (
   Array.isArray(namesOfMenuVisibilities) &&
   namesOfMenuVisibilities.length > 0 &&
   namesOfMenuVisibilities.every(
-    nameOfMenuVisibility => menuVisibilities[nameOfMenuVisibility] === true
+    nameOfMenuVisibility =>
```

Comment:
What's the reasoning behind the change? I think that's also why the prop becomes non-required, right?

Reply:
Yes, it's an optional prop, the prop types were incorrect